source/_integrations/dynalite.markdown

```diff
@@ -109,12 +133,37 @@ area:
                   description: Name of the channel.
                   required: false
                   type: string
-                  default: \"AREA_NAME Channel CHANNEL_NUMBER\"
+                  default: AREA_NAME Channel CHANNEL_NUMBER
+                type:
+                  description: Type of entity this should appear as. Can be either 'light' or if this is a device that is not a light (e.g. water heater), can be 'switch'
+                  require: false
+                  type: string
+                  default: light
                 fade:
                   description: Fade time for the channel, in seconds.
                   required: false
                   type: float
                   default: 2.0
+preset:
+  description: Default presets for any area without the **nodefault** option
+  required: false
+  type: map
+  keys:
+    PRESET_NUMBER:
+      description: The Dynalite preset number in the area.
+      required: true
+      type: map
+      keys:
+        name:
+          description: Name of the preset. When used in an area, it will be 'AREA_NAME name'. For example, if a room's name is "Kitchen" and preset 4 is defined with name "Off", it will appear in HA as **Kitchen Off**          
```

Comment:
```suggestion
          description: "Name of the preset. When used in an area, it will be 'AREA_NAME name'. For example, if a room's name is 'Kitchen' and preset 4 is defined with the name `Off`, it will appear in HA as 'Kitchen Off'."
```